e2e/utils/api.ts

```diff
@@ -22,13 +24,50 @@ export class API {
             body: JSON.stringify(body),
             headers: API.headers,
             resolveWithFullResponse: true
-        }
-        const response = await rp(options)
-        await API.commitUserTransaction(body)
+        };
+        const response = await rp(options);
+        await API.commitUserTransaction(body);
+
+        return response.statusCode;
+    }
+
+    private static async getListingRequestById(id: string) {
+        await API.login();
+        const options = {
+            method: 'GET',
+            uri: `${API.baseUrl}/hearing/${id}`,
+            headers: API.headers,
+            resolveWithFullResponse: true
+        };
+        const response = await rp(options);
+        const responseBody = JSON.parse(response.body);
+        return responseBody;
+    }
+
+    private static async deleteListingRequestWithBody(body: DeleteListingRequestBody) {
+        await API.login();
+        const options = {
+            method: 'POST',
+            uri: `${API.baseUrl}/hearing-part/delete`,
+            body: JSON.stringify(body),
+            headers: API.headers,
+            resolveWithFullResponse: true
+        };
+        const response = await rp(options);
 
         return response.statusCode;
     }
 
+    static async deleteListingRequest(id: string) {
```

Comment:
private?

Reply:
this is used in the test afterAll to cleanup the environment